packages/nextjs/src/config/types.ts

```diff
@@ -133,6 +133,13 @@ export type UserSentryOptions = {
    * Tree shakes Sentry SDK logger statements from the bundle.
    */
   disableLogger?: boolean;
+
+  /**
+   * Automatically create cron monitors in Sentry for your Vercel Cron Jobs if configureed via`vercel.json`.
+   *
+   * Defaults to `true`.
+   */
+  automaticVercelMonitors?: boolean;
```

Comment:
l: should we make specific routes opt-out like we do with `excludeServerRoutes`?